components/ApplicationProviders/ApplicationProviders.tsx

```diff
@@ -27,6 +28,15 @@ const CHAINS: Chain[] = [
   { ...chain.polygon, name: 'Polygon' },
 ];
 
+const Disclaimer: DisclaimerComponent = ({ Text, Link }) => (
+  <Text>
+    By connecting your wallet, you agree to the{' '}
```

Comment:
paging Dr. @jringenberg for wording

Reply:
A very solid boilerplate! I like it. Let's say "Backed" in there specifically so it doesn't seem like rainbowkit's ToS, maybe  slip it in like "...you agree to the **Backed Terms of Service** and acknowledge you have read and understand Backed's **Privacy Policy**."

Reply:
Updated, also took privacy policy out in favor of protocol disclaimer